### PR TITLE
RavenDB-21996 QueryTimings and IRawDocumentQuery/IAsyncRawDocumentQuery documentation

### PR DESCRIPTION
**Pull request by Lwiel**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-21996

### Additional description

Documentation update

### Type of change

- [ ] Bug fix
- [ ] Regression bug fix
- [ ] Optimization
- [ ] New feature

### How risky is the change?

- [x] Low 
- [ ] Moderate 
- [ ] High
- [ ] Not relevant

### Backward compatibility

- [x] Non breaking change
- [ ] Ensured. Please explain how has it been implemented?
- [ ] Breaking change
- [ ] Not relevant

### Is it platform specific issue?

- [ ] Yes. Please list the affected platforms.
- [x] No

### Documentation update

- [ ] This change requires a documentation update. Please mark the issue on YouTrack using `Documentation Required` tag.
- [x] No documentation update is needed 

### Testing by Contributor

- [ ] Tests have been added that prove the fix is effective or that the feature works
- [ ] Internal classes added to the test class (e.g. entity or index definition classes) have the lowest possible access modifier (preferable `private`) 
- [x] It has been verified by manual testing

### Testing by RavenDB QA team

- [ ] This change requires a special QA testing due to possible performance or resources usage implications (CPU, memory, IO). Please mark the issue on YouTrack using `QA Required` tag.
- [x] No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- [ ] Yes. Please list the affected features/subsystems and provide appropriate explanation
- [x] No

### UI work

- [ ] It requires further work in the Studio. Please mark the issue on YouTrack using `Studio Required` tag.
- [x] No UI work is needed

### CODE REVIEW

File: src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs [hunk 1]
```diff
@@ -5,10 +5,19 @@
 
 namespace Raven.Client.Documents.Queries.Timings
 {
+    /// <summary>
+    ///     Representation of query timings.
+    /// </summary>
     public sealed class QueryTimings : IFillFromBlittableJson, IDynamicJson
     {
+        /// <summary>
+        ///     Total time spent on query execution in milliseconds.
```
Comment (arekpalinski) [inline src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs]: This is server side execution, right? Maybe we should say that explicitly?

File: src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs [hunk 2]
```diff
@@ -5,10 +5,19 @@
 
 namespace Raven.Client.Documents.Queries.Timings
 {
+    /// <summary>
+    ///     Representation of query timings.
+    /// </summary>
```
Comment (arekpalinski) [inline src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs:11]: Maybe let's elaborate more what exactly is included there. For example in the docs (https://ravendb.net/docs/article-page/6.0/Csharp/client-api/session/querying/debugging/query-timings) we say:

> detailed stats of the time spent by RavenDB on each part of the query

Also referencing  that docs article would be nice

File: src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs (hunk 1 above)
Comment (Lwiel) [reply src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs]: Yes, I added this to documentation

File: src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs (hunk 2 above)
Comment (Lwiel) [reply src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs:11]: Changed, I think it's enough to say this contains total time spent on server-side query, and time spent on each particular query part

**Review by arekpalinski — APPROVED** [review]

**Pull request merged**